examples/express/package.json

```diff
@@ -38,7 +38,7 @@
   },
   "dependencies": {
     "@opentelemetry/exporter-jaeger": "^0.24.0",
-    "@opentelemetry/instrumentation-express": "^0.24.0",
+    "@opentelemetry/instrumentation-express": "^0.23.0",
```

Comment:
Would it make sense to use `^0.25.0` here?

Reply:
Can't. The released version of @splunk/otel@0.12 requires it to be `0.23.x`.